content/en/storage-providers/setup/setup.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Setup"
```

Comment:
Maybe rename this to `Initialize` to keep things in-line with what was [outline here](https://github.com/filecoin-project/lotus-docs/issues/77#issuecomment-1091139591). Or what do you think?

Reply:
Earlier plan was to initialize the miner in that page and move start to operate section. But if you remember our discussion, we decided to leave it as setup since we will be keeping everything in the same page and moving pre-requisites to another. What do you think?

Reply:
I agree about the content in the page. Was mostly about the wording, and having a `setup`-page within the `setup` section that seemed a bit repetitive/wrong. Maybe some English native speakers have some input here.